packages/@lwc/engine-core/src/framework/rendering.ts

```diff
@@ -610,19 +610,36 @@ function patchElementPropsAndAttrs(
 }
 
 function applyStyleScoping(elm: Element, owner: VM, renderer: RendererAPI) {
+    // TODO [#2762]: this dot notation with add is probably problematic
+    // probably we should have a renderer api for just the add operation
```

Comment:
I think this comment refers to the code on line 620 (`getClassList(elm).add(scopeToken)`). It doesn't particularly make sense to me when applied to `const { getClassList } = renderer;` since there's no dot notation with add being used here.

Reply:
Done! a468b0384